@hubblecommerce/hubble/core/components/category/ProductListingPagination.vue

```diff
@@ -1,20 +1,12 @@
 <template>
     <div class="pagination-wrp">
-        <button @click="selectPage(1)" :disabled="page === 1" class="btn btn-icon">
```

Comment:
Even if its not in design we should keep the min max navigation arrows. 
Replace two single icons with one double arrow icon.